README.md

```diff
@@ -224,6 +224,16 @@ To request automatic tracing support for a module not on this list, please [file
 |----------------------------------------------------------|-----------------------------------------------------------------------------------------|
 | [@opentelemetry/shim-opentracing][otel-shim-opentracing] | OpenTracing shim allows existing OpenTracing instrumentation to report to OpenTelemetry |
 
+## Upgrade guidelines
+
+### 0.14.0 to 0.15.0
```

Comment:
It would be nice if you can update the examples in the same PR (ex: examples/http/server.js)

Reply:
Examples are already broken. e.g. they use the `parent` option of `startSpan` which has been removed a while ago. Therefore I assumed there will be anyway a followup to update/remove/replace all samples.
I will try if I can get them back into a better shape.

Reply:
Updated all samples.